packages/api/src/schema/api-schema.yaml

```diff
@@ -1000,11 +986,10 @@ components:
                   properties:
                     spec:
                       oneOf:
-                        - type: "null"
+                        - type: object
                         - $ref: >-
                             #/components/schemas/asset/properties/storage/properties/ipfs/properties/spec
                 - type: boolean
-                - type: "null"
```

Comment:
@victorges can you please confirm if this is correct?

Reply:
This is official JSON schema syntax: https://json-schema.org/understanding-json-schema/reference/null.html

Do you know why does these OpenAPI tools not support it? Either way, we probably need to do more here to actually represent a field that can only be `null`.